presto-verifier/src/test/java/com/facebook/presto/verifier/checksum/TestChecksumValidator.java

```diff
@@ -297,9 +304,9 @@ public void testArray()
         ChecksumResult controlChecksum = new ChecksumResult(
                 5,
                 ImmutableMap.<String, Object>builder()
-                        .put("int_array_checksum", new SqlVarbinary(new byte[] {0xa}))
+                        .put("int_array_checksum", new SqlVarbinary(new byte[]{0xa}))
```

Comment:
Your formatting is off. Please follow 7e and 7f  in https://our.internmc.facebook.com/intern/wiki/DataInfra/Presto/Source/
to install Airlift code style, and then format the file with (option + cmd + L).